scripts/publish.sh

```diff
@@ -18,10 +18,19 @@ scripts=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 cd "${scripts}"
 
 IMAGE_SHA_MATCHED="FALSE"
-AWS_FOR_FLUENT_BIT_VERSION=$(cat ../AWS_FOR_FLUENT_BIT_VERSION)
+# TODO: Make it work for both build versions (or move entire logic in this script)
+if [ -z "$BUILD_VERSION" ]; then
+    AWS_FOR_FLUENT_BIT_VERSION=$(cat ../AWS_FOR_FLUENT_BIT_VERSION)
```

Comment:
Defaulting to current behavior in publish.sh when BUILD_VERSION is not set. This will be removed eventually.